src/components/Cart/Cart.module.scss

```diff
@@ -8,65 +8,80 @@
 
 .cart-item {
   display: flex;
-  border-bottom: 2px solid #f46805;
-  padding: 1rem 0;
+  // border-bottom: 2px solid #f46805;
+  // padding: 1rem 0;
   margin: 1rem 1.5rem;
   gap: 20px;
+  box-shadow: 0 4px 16px rgb(23 23 24 / 15%);
+  border-radius: 10px;
 }
 
 .cart-item-img {
-  width: 168px;
-  height: 92px;
+  width: 145px;
+  height: 80px;
 }
 
 .cart-item-img img {
```

Comment:
read about some css naming strategies for css modules in react, consider BEM approach or just simple naming convention for css classes.